roles/kubernetes-apps/ansible/tasks/netchecker.yml

```diff
@@ -4,7 +4,7 @@
     dest: "{{kube_config_dir}}/{{item.file}}"
   with_items:
     - {file: netchecker-agent-ds.yml, type: ds, name: netchecker-agent}
-    - {file: netchecker-agent-hostnet-ds.yml, type: ds, name: netchecker-agent-hostnet}
+    - {file: netchecker-agent-hostnet-ds.j2, type: ds, name: netchecker-agent-hostnet}
```

Comment:
we name our templates {{ original_filename + ".j2" }}
If you're renaming it, can you change it to netchecker-agent-hostnet-ds.yml.j2?